### PR TITLE
Fix typos plus clarifications

### PR DESCRIPTION
**Pull request by TorstenScheck**
Thank you very much for all the Gotcha's which I experienced during your course.

Here are some typos I found and some clarification which would have helped me.
Your text is already meeting very high standards, though.

Please merge or cherry pick at your discretion.

### CODE REVIEW

File: src/part05.rs [hunk 1]
```diff
@@ -47,7 +47,8 @@ impl BigInt {
         }
     }
 
-    // We can convert any vector of digits into a number, by removing trailing zeros. The `mut`
+    // Any vector of digits, which meets the structure of BigInt's `data` field, can be easily
```
Comment (RalfJung) [inline src/part05.rs:50]: This makes it sound like conversion is possible only with those specific vectors that do meet the structure, but the entire point here is that we do not have to assume anything about the vector.

Could you elaborate on what you found confusing here? Maybe that helps us to find a better wording.

Comment (TorstenScheck) [reply src/part05.rs:50]: When I did Exercise 05.1 it was somehow natural for me to treat my Vec<u64> as big endian, e.g.  vec![1,2,0] for 0\*64^0 + 2\*64^1 + 1\*64^2. So my from_vec() implementation had to fill the BigInt data in reverse, whereas your implementation just copied the complete vector into the struct's data field. The examples and unit tests later in the course wouldn't fit with my implementation. No big deal, but maybe we could give a hint for future learners.

Comment (RalfJung) [reply src/part05.rs:50]: Okay, so maybe the wording should be something like "We can convert any little-endian vector of digits (i.e., least-significant digit first)" -- would that have helped?

Comment (TorstenScheck) [reply src/part05.rs:50]: Sure, that would have led me into the right direction.

Thank you for still maintaining this tutorial despite your busy work. I've read about your accomplishment during the all hands week. Keep on with your great work!

Comment (RalfJung) [reply src/part05.rs:50]: > I've read about your accomplishment during the all hands week. Keep on with your great work!

Thanks! Watch our for a blog post on this topic. :)

**Pull request merged**